clojure-mode.el

```diff
@@ -2684,6 +2684,19 @@ lists up."
     (insert sexp)
     (clojure--replace-sexps-with-bindings-and-indent)))
 
+(defun clojure--collect-ns-aliases ()
```

Comment:
I guess this one can even be public API and optionally take a `ns` form as a param.

Reply:
Do you mean having it as a string->list<string> pure function? Or taking a `point` param marking the location of the start of the ns form?

Reply:
Pure function taking a ns form as param.